app/views/devise/sessions/new.html.slim

```diff
@@ -12,4 +12,5 @@
       = render 'devise/sessions/form'
       h2.m0.py3.text-over-line
         span or use your
-      button.btn.btn-block.btn-primary.bg-navy.mb1 DS Logon
+      = link_to '<button class="btn btn-block btn-primary bg-navy mb1">DS Logon</button>'.html_safe, '/users/auth/saml'
+
```

Comment:
I believe the convention is to use `button_to`. Check out `app/views/idp/index.html.slim` for an example usage.